### ISSUE TITLE
(PowerShell) Driver with headless mode does not always clean up chrome.exe after driver.quit()

### ISSUE THREAD

**Issue by adamschachne** [seleniumbase/SeleniumBase#2991]
This only seems to be a problem when running in headless mode. Below is a minimal example to reproduce the issue.

Windows 10 22H2
Google Chrome 127.0.6533.89
seleniumbase==4.29.6

```
from seleniumbase import Driver

driver = Driver(headless=True)
try:
    driver.open("seleniumbase.io/simple/login")
    driver.sleep(5)
finally:
    driver.quit()
```

Run this in Windows Powershell. Open a task manager and watch the child processes of the powershell window. Roughly once every 10 executions, the chrome.exe process will be orphaned when python finishes. When this happens, chrome.exe will also skyrocket in CPU utilization.

I've tried using headless2=True and chromium_args="--headless=new", but the results are the same. 

plain selenium does not seem to have this bug:

```
from selenium.webdriver import Chrome
from selenium.webdriver.chrome.options import Options

options = Options()
options.add_argument("--headless=new")
driver = Chrome(options=options)
try:
    driver.get("https://seleniumbase.io/simple/login")
    sleep(5)
finally:
    driver.quit()
```

**Comment by mdmintz** [comment]
Thanks for reporting this. I was able to reproduce it with your instructions:

<img src="https://github.com/user-attachments/assets/0248446e-8573-4c04-bc1a-d5f5eb820ff5" title="Powershell memory leak" width="750">

Looks like it's only happening with PowerShell, and intermittently after many runs. No idea why though. The `selenium` `driver.quit()` method isn't always cleaning up the Chrome instances, and that method comes from regular Selenium, (not SeleniumBase), so I'm not sure how to improve the cleanup since it's being called by an external method.

Maybe changing the default SeleniumBase command-line options for PowerShell could help.
Maybe there's a bug with PowerShell causing random conflicts with Selenium.
Maybe it's a bug with regular Selenium if `driver.quit()` isn't always quitting everything.

Let me know if you find more information. I'll try to do some of my own investigation.

**Comment by alish511** [comment]
why on driver object uc_gui_click_x_y method dont work and give this error 'Chrome' object has no attribute 'uc_gui_click_captcha'

**Comment by mdmintz** [comment]
I think it has something to do with the `--no-sandbox` arg being set on Windows.

**Comment by mdmintz** [comment]
This was resolved in `4.29.7` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.29.7
